include/coap/coap_session.h

```diff
@@ -77,6 +77,7 @@ typedef struct coap_session_t {
   coap_tick_t last_rx_tx;
   coap_tick_t last_tx_rst;
   coap_tick_t last_ping;
+  coap_tick_t last_pong;
```

Comment:
In the commit message, you state that the time accuracy is 1ms. This may or may not be true, depending on the system's clock resolution and the value of COAP_TICKS_PER_SECOND.

Reply:
Sure, I understand. But I want to avoid that people read the commit message and get the impression that this 1ms resolution exists (and base assumptions on that).

Reply:
OK. Can you change the message during the merge or do you want me to amend the commit in the pull request ? The latter is not really push-button with git, unlike svn, but I guess I can do it by rebasing.

Reply:
I think, I can change that manually (not push-button as well but not too hard).